apis/nucleus/src/components/Footer.jsx

```diff
@@ -30,15 +29,21 @@ const CellFooter = {
   className: 'njs-cell-footer',
 };
 
-function Footer({ layout, titleStyles = {}, translator }) {
+function Footer({ layout, titleStyles = {}, translator, flags }) {
   const footerStyle = titleStyles.footer;
   const hasFilters = layout?.filters?.length > 0 && layout?.qHyperCube?.qMeasureInfo?.length > 0;
   const filtersFootnoteString = generateFiltersString(layout?.filters ?? [], translator);
   const showFilters = !layout?.footnote && hasFilters && filtersFootnoteString;
+  const newSheetEnabled = flags?.isEnabled('VIZ-7_NEW_SHEET');
 
   return layout && layout.showTitles && (layout.footnote || showFilters) ? (
     <StyledGrid container>
-      <Grid item className={classes.itemStyle} data-testid={CellFooter.className}>
+      <Grid
+        item
+        className={classes.itemStyle}
+        data-testid={CellFooter.className}
+        style={{ paddingTop: (theme) => (newSheetEnabled ? 0 : theme.spacing(1)) }}
```

Comment:
from my testing.
For classic theme (not cards) it should always have spacing.
and for cards theme 0 is to small, maybe 1px?